python/lsst/ctrl/bps/cli/resources.yaml

```diff
@@ -0,0 +1,6 @@
+cmd:
+  import: lsst.ctrl.bps.cli.cmd
+  commands:
+    - prepare
+    - report
+    - submit
```

Comment:
you don't need this file, these commands are available to the `BpsCli` via it's attribute `localCmdPkg = "lsst.ctrl.bps.cli.cmd"`